AmpleOrganics/Blaze.vue
# Vuepress inital setup (for documentation)

Initial setup for Vuepress for our component library documentation

This should allow anyone to build out documentation with instructions from the official [documentation](https://vuepress.vuejs.org/)

# QA/setup
1. install the new packages by running `yarn` 
2. run `yarn docs:dev` to start the dev server!
3. go to the URL built and you should see 'Hello VuePress, this is Blaze'
